changelog.d/datadog_agent_http_header.fix.md

```diff
@@ -0,0 +1,4 @@
+Fix bug in datadog sink where data sent originating from the datadog agent is not 
+interpreted by the backend as an agent message.
```

Comment:
```suggestion
Fix bug in the `datadog_logs` sink where data sent originating from the Datadog Agent is not 
interpreted by the Datadog backends as an Agent message.
```

Reply:
Agree with all of this except not sure "backend" should be pluralized to "backends."